server/index.html

```diff
@@ -7,6 +7,7 @@
   <script src="node_modules/react/dist/react.js"></script>
   <script src="node_modules/react-dom/dist/react-dom.js"></script>
   <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.css">
+  <link rel="stylesheet" href="node_modules/react-redux-toastr/lib/css/react-redux-toastr.css">
```

Comment:
Is this necesarry? Try
`import 'react-redux-toastr/lib/css/react-redux-toastr.css'`
 in appFactory.js